arviz/stats/stats.py

```diff
@@ -40,7 +40,9 @@ def bfmi(energy):
         chain in the trace.
     """
     energy_mat = np.atleast_2d(energy)
-    return np.square(np.diff(energy_mat, axis=1)).mean(axis=1) / np.var(energy_mat, axis=1)
+    return np.square(np.diff(energy_mat, axis=1)).mean(  # pylint: disable=no-member
```

Comment:
This looks very weird, maybe it is better to separate the expression into two (and not just break the lines)